src/prefabs/datePicker.js

```diff
@@ -56,6 +56,77 @@
           key: 'dateFormat',
           type: 'TEXT',
         },
+        {
+          label: 'Use min/max date',
```

Comment:
When the datePicker is used in other prefabs, these options might need to be added in those prefabs as well.